.pre-commit-config.yaml

```diff
@@ -17,7 +17,6 @@ repos:
       exclude: "^examples|^test"
     - id: check-yaml
       # Can't check source yaml since it has go templates in it.
-      # Can't check the examples because example demos contain multiple Kubernetes objects defined
-      # in a single file.
-      exclude: "^helm-charts|^test|^examples"
+      exclude: "^helm-charts"
+      args: [ --allow-multiple-documents ]
```

Comment:
Just noting this is also in https://github.com/signalfx/splunk-otel-collector-chart/pull/946/files

Reply:
yes this PR was built off that branch as it requires this change.